tests/feature/test_tags.py

```diff
@@ -15,13 +15,6 @@ def test():
 
     assert test.__scenario__.tags == set(['scenario_tag_1', 'scenario_tag_2'])
     assert test.__scenario__.feature.tags == set(['feature_tag_1', 'feature_tag_2'])
-
-    assert getattr(test, 'scenario_tag_1')
-    assert getattr(test, 'scenario_tag_2')
-
-    assert getattr(test, 'feature_tag_1')
-    assert getattr(test, 'feature_tag_2')
```

Comment:
Above asserts fail on `pytest>=4.1.0` - but I think that we don't really need those attributes (tags) on `test` function as we don't use them. Correct me if I am wrong.